front/src/hooks/useLoadAndSetGlobalPreferences.ts

```diff
@@ -59,17 +56,17 @@ export const useLoadAndSetGlobalPreferences = () => {
         } else {
```

Comment:
This case and the `NotFoundView` are both triggering redirects. Could you remove this case since we actually do not need it anymore.